substrafl/dependency/manage_dependencies.py

```diff
@@ -194,7 +194,6 @@ def compile_requirements(dependency_list: List[Union[str, Path]], *, dest_dir: P
         "piptools",
         "compile",
         "--resolver=backtracking",
-        "--no-emit-index-url",
```

Comment:
In the end, the param was an error (from myself) cause it might leak some extra index url, but some other might be needed (such as the torch cpu one)